app/assets/javascripts/features/sliderToggle.js

```diff
@@ -0,0 +1,52 @@
+class sliderToggle {
+
+  constructor(element) {
+    this.element = element;
+    this.colors = ['#F2C287', '#823E2A', '#7A2F22', '#5E180A', '#4A0900', '#330600', '#1A0300', '#0F0300'];
```

Comment:
If we had time this would be fun hooking up CSS variables with. But we don't have time.... Maybe next week?